airflow/models/__init__.py

```diff
@@ -43,3 +43,19 @@
 # Must be loaded after loading DAG model.
 # noinspection PyUnresolvedReferences
 import airflow.jobs  # noqa: F401 isort # isort:skip
+
+
+def dag_fileloc_hash(full_filepath: str) -> int:
```

Comment:
We don't put the code in __init__.py. We used to do it and now we have serious problems with cyclical imports.

Reply:
Done